assets/dev/scss/editor/_templates-modal.scss

```diff
@@ -488,7 +488,13 @@ label.elementor-template-library-order-label {
 	}
 }
 
-.elementor-template-library-template-page {
+.elementor-template-library-template-page,
+.elementor-template-library-template-lp {
+
+	.elementor-template-library-template-name {
+		display: initial;
```

Comment:
Why?

Reply:
Because there is page-specific CSS that hides the template title for non-page items, this is to override that.

Reply:
Removed.